modular_skyrat/modules/soupiersoups/code/soup_mixtures.dm

```diff
@@ -0,0 +1,414 @@
+/datum/reagent/consumable/nutriment/soup
+	nutriment_factor = 20 // Upped to counter the loss of random ingredients.
+	//Some soups will create between 30-60u to counter the loss of additional reagents.
+
+/datum/chemical_reaction/food/soup
+	ingredient_reagent_multiplier = 0.0
+	percentage_of_nutriment_converted = 1.0
```

Comment:
I applied it repeatedly here because when I adjusted the parent soup, it did not apply to all, even if they didn't have individual modifiers. It was the only way I could ensure they all followed the same pattern.

Reply:
Yeah, I'm looking into how the base reaction works and instead seeing if I can make the subtype consume the usual offenders in the soup.